### ISSUE TITLE
Failed to play sound /sounds/FreedesktopSounds/service-login.mp3 DOMException

### ISSUE THREAD

**Issue by SpaceboyRoss01** [os-js/OS.js#700]
## Expected behaviour

I'm trying to do start OS.js.

## Actual behaviour

But the startup sound doesn't play happens instead.

## Steps to reproduce the error

1. Launch OS.js V3 in Chromium

**Comment by andersevenrud** [comment]
https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

> Chrome's autoplay policies will change in April of 2018 and I'm here to tell you why and how this is going to affect video playback with sound. 

> Chrome's autoplay policies are simple:
> Autoplay with sound is allowed if: User has interacted with the domain (click, tap, etc.).

So basically, this should not be an issue when using a login form because the user will do interaction before sound is played.

**Comment by SpaceboyRoss01** [comment]
It doesn't work with autologin.20.12.2018, 13:43, "Anders Evenrud" <notifications@github.com>:https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

Chrome's autoplay policies will change in April of 2018 and I'm here to tell you why and how this is going to affect video playback with sound.


Chrome's autoplay policies are simple:
Autoplay with sound is allowed if: User has interacted with the domain (click, tap, etc.).

So basically, this should not be an issue when using a login form because the user will do interaction before sound is played.

**Comment by andersevenrud** [comment]
> It doesn't work with autologin

Yes, exactly.
